api/admin/controller.py

```diff
@@ -530,6 +538,14 @@ def media(self):
         """Return the supported media types for a work and their schema.org values."""
         return Edition.additional_type_to_medium
 
+    def rights_status(self):
+        """Return the supported rights status values with their names and whether
+        they are open access."""
+        return {uri: dict(name=name,
+                          open_access=(uri in RightsStatus.OPEN_ACCESS),
+                          allows_derivatives=(uri in RightsStatus.ALLOWS_DERIVATIVES))
+                for uri, name in RightsStatus.NAMES.iteritems()}
```

Comment:
Only rights statuses that allow derivatives are selectable for a new cover image.

Reply:
This could be enforced on the data model level but I think this is the right level; we should leave room for someone modifying an image that they own the rights to.

Reply:
This is likely to be a real case for works published through the library -- the author wants a photo they own to be used as the cover. Maybe we don't need to handle that yet, but "permission granted" is a reason why a RightsStatus that wouldn't ordinarily belong here should go here anyway.

Reply:
Yeah, I'd like to handle that case but I'm not sure how to make it clear in the UI.

Reply:
Maybe just let 'In copyright' be an option. The rights_explanation will explain why it's okay to make a derivative work. That's the reason you'd need a rights_explanation, right? When it's not obvious from the RightsStatus.

Reply:
I was thinking it was more to justify why the work is open access in case it's challenged later.

Reply:
OK, the rights_explanation on the original resource would have the reason why the image is open access, and the rights_explanation on the derivative would have the reason why the derivative could be made from a non-open-access original.

Again, it's not necessary to implement this in this branch, I just want to make sure we don't exclude the possibility.

Reply:
Yeah, that would work fine in the data model but it would make the UI complicated. For now I'm just using the same rights explanation for both.

Reply:
I think the UI wouldn't need to change. You would have the standard free licenses and 'other', plus a box to write an explanation. If you choose one of the standard licenses then your explanation is associated with the original resource, and the derivative explanation is 'the original resource was licensed for this'. Otherwise the explanation is associated with the derivative resource as 'why did we do this'.

Reply:
If you choose 'other', is the rights status unknown?

Reply:
I added 'in copyright' and 'other' as options, and made it save the rights explanations differently depending on the URI.